public/leaderboardList.php

```diff
@@ -245,28 +214,32 @@ function UpdateLeaderboard(lbID) {
         echo "<tr>";
         echo "<td style='width:10%;'>Start:</td>";
         echo "<td>";
-        echo "<input type='text' id='LB_" . $lbID . "_Mem1' value='$memStart' style='width: 100%;' " . ($editAllowed ? "" : "readonly") . "/>";
+        echo "<p>$memStart</p>";
```

Comment:
Could probably just remove the definition dumps here, as they're provided more explicitly in the leaderboard page itself:

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/c3d6d642-cf51-48d6-b681-98e621d92e3a)

Or at least hide them by default like the Code fields in the Manage Achievements page.

Reply:
Removed the definitions in latest. Also made a few extra QoL tweaks:

![Screenshot 2023-09-17 at 10 19 34 AM](https://github.com/RetroAchievements/RAWeb/assets/3984985/1599b15c-31d2-4dfc-9a5f-9488e5e29a9b)

Alternating colors now works based on the leaderboard (now the explicit `<tr>`), and the page does a hard reload after a successful save.

Reply:
Probably don't need the code notes side panel on this page any longer.

Reply:
Yep, you're probably right. Removed in https://github.com/RetroAchievements/RAWeb/pull/1873/commits/c9ff1bfbb4cb45a0b175b4d0ca20e79d4e123da8.